_layouts/person.html

```diff
@@ -141,8 +141,8 @@ <h1 class="person-name">{{ person.title | escape_once }}</h1>
   <section class="person-shows">
     {% for record in person.shows %}
     <div class="person-show">
-      <div class="person-show-poster">
-        <a href="{{ record.item.url }}" title="{{ record.item.title | escape_once }}">
+      <a href="{{ record.item.url }}" class="person-show-title" title="{{ record.item.title | escape_once }}">
```

Comment:
I'll check that out. Made it .person-show-title as otherwise the underline would hilariously change colour depending on whether the hover was on the poster or the text. Will see if .person-show works too.